gradebookng/tool/src/java/org/sakaiproject/gradebookng/GradebookNgApplication.properties

```diff
@@ -375,7 +375,7 @@ column.header.coursegradeoverride.gradeoverride = Grade Override
 coursegrade.display.percentage-first = {0}
 coursegrade.display.percentage-second = ({0})
 coursegrade.display.points-first = {0}/{1}
-coursegrade.display.points-second = [{0}/{1}]
+coursegrade.display.points-second = <br>[{0}/{1}]
```

Comment:
I noticed this after I merged. Dodgy as, but totally fine :+1: Saves adding a new label etc etc.